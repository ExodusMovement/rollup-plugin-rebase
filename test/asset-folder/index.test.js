import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Asset Folder", async () => {
  await bundle(root, "index.js", "output/index.js", { folder: "bundled-assets" })

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await list(root, "output/bundled-assets")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/bundled-assets/gToHHcDx.css")).toMatchSnapshot()

  await clean(root, "output")
})
