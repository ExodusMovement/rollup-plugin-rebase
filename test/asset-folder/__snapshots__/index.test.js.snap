// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Asset Folder 1`] = `
"bundled-assets
index.js"
`;

exports[`Asset Folder 2`] = `
"bXipeBrs.png
bsdudxaF.md
gToHHcDx.css
gToHHcDx.css.map"
`;

exports[`Asset Folder 3`] = `
"import content from './bundled-assets/bsdudxaF.md';
import css from './bundled-assets/gToHHcDx.css';

function index() {
  return content + css
}

export default index;
"
`;

exports[`Asset Folder 4`] = `
"body {
  font-size: 20px;
  background: url(./bXipeBrs.png) top left repeat;
}

/*# sourceMappingURL=gToHHcDx.css.map */"
`;
